libcxx/test/std/utilities/debugging/is_debugger_present_with_debugger_lldb.sh.cpp

```diff
@@ -0,0 +1,59 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20, c++23
+// REQUIRES: host-has-lldb-with-python
+// The Android libc++ tests are run on a non-Android host, connected to an
+// Android device over adb.
+// UNSUPPORTED: android, linux && no-filesystem && no-localization
+// XFAIL: LIBCXX-AIX-FIXME, LIBCXX-PICOLIBC-FIXME
```

Comment:
```suggestion
// XFAIL: LIBCXX-PICOLIBC-FIXME
```
@daltenty Q: Did you miss removing this or did you leave it on purpose? I believe lldb should work on AIX CI, so I'll remove it.